tgui/packages/tgui/interfaces/TransceptionInterlink/index.tsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * @file
+ * @copyright 2024
+ * @author glowbold (https://github.com/pgmzeta)
+ * @license MIT
+ */
+
+import { useBackend } from '../../backend';
+import { Box, Button, Section, Stack } from '../../components';
+import { Window } from '../../layouts';
+import { TransceptionInterlinkData } from './type';
+
+export const TransceptionInterlink = (_props, context) => {
+  const { data, act } = useBackend<TransceptionInterlinkData>(context);
+  const { pads, crate_count } = data;
+  return (
+    <Window title="Transception Interlink" height="340" width="400">
+      <Window.Content>
+        <Stack vertical fill>
+          <Stack.Item>
+            <Section>
+              <Stack>
+                <Stack.Item width="100%" fontSize="1.2em">
```

Comment:
Rather than `width="100%"`, put a `justify="space-between"` on the `Stack` above this.